internal/cmd/grants.go

```diff
@@ -58,46 +58,62 @@ func newGrantsListCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
-			type row struct {
-				Identity string `header:"IDENTITY"`
+			type userTable struct {
```

Comment:
These should probably be userRow{} and groupRow{} given that you're creating a slice of both (not a slice of tables)